sys/vm/vm_mmap.c

```diff
@@ -1488,6 +1490,12 @@ kern_mincore(struct thread *td, uintptr_t addr0, size_t len,
 				    pmap_is_referenced(m) ||
 				    (m->a.flags & PGA_REFERENCED) != 0)
 					mincoreinfo |= MINCORE_REFERENCED_OTHER;
+
+				/* Expose capability tracking information */
+				mincoreinfo |=
+				    ((mas.flags & PGA_CAPSTORE) != 0) ?
+					  MINCORE_CAPSTORE :
+					  0;
```

Comment:
```suggestion
				if ((mas.flags & PGA_CAPSTORE) != 0)
					mincoreinfo |= MINCORE_CAPSTORE;
```
(maybe it was more complicated before?)

Reply:
Though given the comments above I wonder if you should be using `m->a.flags` rather than a cached value?

Reply:
`mincore` is inherently race-y, so I'm a little surprised they do that dance, but you're probably right that this should be `m->a.flags` in keeping with upstream.  Another possibility would be to write something like

```
if (mas & PGA_REFERENCED) != 0 ||
    pmap_is_referenced(m) ||
    (m->a.flags & PGA_REFERENCED) != 0)
// ...

if ((mas & PGA_CAPSTORE) != 0)
// ...
```

but that seems excessive.